app/scenes/Invite.js

```diff
@@ -26,15 +27,16 @@ type Props = {|
 type InviteRequest = {
   email: string,
   name: string,
+  role: string,
```

Comment:
```suggestion
  role: "member" | "viewer" | "admin",
```

lets type this a little more strictly, there's a few spots in this PR